build/factory/memory-server-factory.js

```diff
@@ -0,0 +1,108 @@
+"use strict";
```

Comment:
You can hide these in diffs by creating a `.gitattributes` file with these contents:

```
build/* linguist-generated=true
build/* -diff
```